<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>SoSanh</string>
    </property>
   </widget>
   <widget class="QLabel" name="compoF">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>So sánh file</string>
    </property>
   </widget>
   <widget class="QLabel" name="dataF2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>461</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Nhập dữ liệu cho File 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editF2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="RetureF">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Kết quả</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>360</y>
      <width>251</width>
      <height>28</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>461</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>File 1 nội dung là &quot;chung toi la chien si, tieu doan 307 anh hung&quot;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
